Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_24.md

```diff
@@ -0,0 +1,12 @@
+
+#### Mappers
+
+##### CrowdStrike Falcon Mapper
+
+- Mapped the new field **Behaviour Pattern Disposition Details**.
```

Comment:
```suggestion
Mapped the new field **Behaviour Pattern Disposition Details**.
```

Is that how you are spelling Behaviour?  It should be Behavior.  With the u is a British spelling.

Reply:
other fields in the content pack are spelled *behaviour*, I thought I should follow the convention.